### PR TITLE
[Scope change] Revisions

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Components/Sidesheet/SidesheetWrapper/RevisionForm.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                        </>
+                    )}
+                </>
+            </ButtonContainer>
+        </ActionBar>
+    );
+};
+
+const WarningRevisionBanner = () => {
+    return (
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Sidesheet/SidesheetWrapper/RevisionForm.tsx:140]: File

File: src/apps/ScopeChangeRequest/Components/Sidesheet/SidesheetWrapper/RevisionForm.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    flex-direction: column;
+    overflow-y: scroll;
+    overflow-x: hidden;
+`;
+
+interface SubmitActionBarProps {
+    cancel: () => void;
+}
+
+export const SubmitActionBar = ({ cancel }: SubmitActionBarProps): JSX.Element => {
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Components/Sidesheet/SidesheetWrapper/RevisionForm.tsx:96]: Burde kansje vært i egen fil ?

File: src/apps/ScopeChangeRequest/Components/Sidesheet/SidesheetWrapper/SidesheetWrapper.tsx [hunk 3]
```diff
@@ -52,34 +57,43 @@ export function SidesheetWrapper({ item, actions }: SidesheetWrapperProps): JSX.
...
+            </div>
+            <Switch>
+                <Case when={editMode}>
+                    <ScopeChangeRequestEditForm />
+                </Case>
+                <Case when={revisionMode}>
+                    <RevisionForm cancel={() => setRevisionMode(false)} />
+                </Case>
+                <Case when={true}>
+                    <div>
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Components/Sidesheet/SidesheetWrapper/SidesheetWrapper.tsx:71]: egen komponent? linje 71  til 94

**Comment by Noggling** [comment]
enkelte filer trenger en opprydding

**Review by Noggling — APPROVED** [review]

**Pull request merged**